iep-platformservice/src/main/java/com/netflix/iep/platformservice/PropertiesReader.java

```diff
@@ -78,4 +83,22 @@
       throw new IOException("request failed with status: " + response.status());
     }
   }
+
+  /**
+   * Update the {@link ConfigManager#dynamicConfigManager()} with the properties. The value
+   * for a special key {@code netflix.iep.override} will be treated as a Typesafe Config string
+   * so that all constructs can be supported. Other properties will get used directly.
+   */
+  void updateDynamicConfig(Properties props) {
```

Comment:
It feels like the README for the `iep-platformservice` module needs some updates to clarify when or when not to use it, given where we are going with this commit.

Reply:
You don't use it directly, it is just the shared logic for `iep-config` and `iep-module-archaius2`. This hook is just to ensure that the dynamic config usage will work when using one of those old modules to ease gradual transition if you cannot move right away.

Reply:
Fair point. Note that the existing README still references the archaius2 module. :)